base/applications/cleanmgr/precomp.h

```diff
@@ -0,0 +1,152 @@
+/*
+ * PROJECT:         ReactOS Disk Cleanup
+ * LICENSE:         GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:         Precompiled header file
+ * COPYRIGHT:       Copyright 2020 Arnav Bhatt (arnavbhatt288 at gmail dot com)
+ */
+ 
+#ifndef _CLEANMGR_PRECOMP_H
+#define _CLEANMGR_PRECOMP_H
+ 
+#define COBJMACROS
+#define ONLY_PHYSICAL_DRIVE 3
+#define ARR_MAX_SIZE 512
+
+#include "resource.h"
+
+#include <windows.h>
+#include <windowsx.h>
+#include <wchar.h>
+#include <shlwapi.h>
+#include <commctrl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <strsafe.h>
+#include <string.h>
+#include <guiddef.h>
+#include <inttypes.h>
+#include <debug.h>
+#include <shlobj.h>
+#include <shobjidl.h>
+#include <shlguid.h>
+#include <uxtheme.h>
+#include <vsstyle.h>
+#include <wininet.h>
+#include <cpl.h>
+#include <winreg.h>
+
+typedef struct
+{
+    uint64_t TempSize;
+    uint64_t RecycleBinSize;
+    uint64_t ChkDskSize;
+    uint64_t RappsSize;
+    uint64_t CountSize;
+} DIRSIZE;
+
+typedef struct
+{
+    HWND hChoicePage;
+    HWND hOptionsPage;
+    HWND hSagesetPage;
+    HWND hTab;
+} DLG_VAR;
+
+typedef struct
+{
+    WCHAR DriveLetter[ARR_MAX_SIZE];
+    WCHAR TempDir[ARR_MAX_SIZE];
+    WCHAR RappsDir[MAX_PATH];
+} WCHAR_VAR;
+
+typedef struct
+{
+    BOOL TempClean;
+    BOOL RecycleClean;
+    BOOL ChkDskClean;
+    BOOL RappsClean;
+    BOOL SysDrive;
+} BOOL_VAR;
+
+DIRSIZE sz;
+DLG_VAR dv;
+WCHAR_VAR wcv;
+BOOL_VAR bv;
```

Comment:
WCHAR_VAR.
BOOL_VAR.

Why all these globals, why the arbitrary choice of combining them per type (and using those cryptic names)?